ragnarok_packets/src/position.rs

```diff
@@ -52,10 +57,18 @@ impl WorldPosition2 {
     }
 
     pub fn to_origin_destination(self) -> (WorldPosition, WorldPosition) {
-        (WorldPosition { x: self.x1, y: self.y1 }, WorldPosition {
-            x: self.x2,
-            y: self.y2,
-        })
+        (
+            WorldPosition {
+                x: self.x1,
+                y: self.y1,
+                direction: 0,
```

Comment:
Does this packet not have a direction in it? Please have a look at the `from_bytes()` function for this packet. It seems it was planned to read a direction there too. If that is not the case, then please remove that comment in there.

Reply:
I have tried to understand the reasoning, but seems not related with direction.